src/int.cr

```diff
@@ -666,18 +666,16 @@ struct Int64
 end
 
 struct Int128
-  # TODO: eventually update to literals once UInt128 bit support is finished
-  MIN = new(1) << 127
-  MAX = ~MIN
+  MIN = -170141183460469231731687303715884105728_i128
+  MAX =  170141183460469231731687303715884105727_i128
```

Comment:
Would be easy to have a character off here. Is there a spec asserting these numbers are `2**127 - 1` and `-(2**127)`?

Reply:
wouldn't it be better if those numbers are calculated from macros, sth like:
```cr
MIN = {{ -(2 ** 127) }}
MAX = {{ 2 ** 127 - 1 }}
```
?

Reply:
Quite possible, these numbers were taken from rust's 128-bit integer support [RFC](https://github.com/rust-lang/rust/blob/master/src/doc/unstable-book/src/language-features/i128-type.md) - if they indeed turn out to be wrong, linked wiki page could use an update too.